service/cloudconfig/cloud_config.go

```diff
@@ -52,7 +53,7 @@ func New(config Config) (*CloudConfig, error) {
 
 // NewMasterTemplate generates a new worker cloud config template and returns it
 // as a base64 encoded string.
-func (c *CloudConfig) NewMasterTemplate(customObject kvmtpr.CustomObject, certs certificatetpr.AssetsBundle, node clustertprspec.Node) (string, error) {
+func (c *CloudConfig) NewMasterTemplate(customObject kvmtpr.CustomObject, certs certificatetpr.AssetsBundle, node clustertprspec.Node, keys map[randomkeytpr.Key][]byte) (string, error) {
```

Comment:
Here we see how inconsistent the implementation of the keys is. We have a kind of clean type `certificatetpr.AssetsBundle` for the certs. It would be nice to have something similar for the keys. Lets create an issue for this to refactor at some point.

Reply:
Will make it